grakn-engine/src/test/java/ai/grakn/engine/controller/api/EntityTypeControllerTest.java

```diff
@@ -134,11 +148,6 @@ public void assignAttributeToEntityTypeShouldExecuteSuccessfully() {
 //
```

Comment:
Is the functionality for these commented tests not available yet? I thought they were.

Reply:
also fixed in ddc298e472a40a428129c4df7c7cfa6c75510c0a

i'll run over the changes while the test are running